resources/js/views/meetings/Index.vue

```diff
@@ -0,0 +1,224 @@
+<template>
+  <b-container class="mt-3 mb-5">
+    <b-row class="mb-3">
+      <b-col>
+        <h2>
+          {{ $t('meetings.currentlyRunning') }}
+        </h2>
+      </b-col>
+      <b-col sm='12' md='3'>
+        <b-input-group>
+          <b-form-input @change="$root.$emit('bv::refresh::table', 'meetings-table')" :disabled="isBusy || loadingError" ref="search" :placeholder="$t('app.search')" v-model="filter"></b-form-input>
+          <b-input-group-append>
+            <b-button @click="$root.$emit('bv::refresh::table', 'meetings-table')" :disabled="isBusy || loadingError" variant="success"><b-icon icon="search"></b-icon></b-button>
+          </b-input-group-append>
+        </b-input-group>
+      </b-col>
+    </b-row>
+    <hr>
+
+    <b-overlay  :show="loadingError" >
+
+      <template #overlay>
+        <div class="text-center my-2">
+          <b-spinner v-if="isBusy" ></b-spinner>
```

Comment:
This does not make sense, since the overlay is only shown if there is an loading error!

Reply:
The loadingError is set to false if the requsted was successfull. During the request the loadingError is still true and isBusy true

Reply:
But in this case there are two spinner one in the table and one in the overlay!
![grafik](https://user-images.githubusercontent.com/6422711/109703201-79e00480-7b95-11eb-9ddb-b5261e9b8e98.png)